jest-transform.cjs

```diff
@@ -0,0 +1,12 @@
+const path = require("path");
```

Comment:
Why is this needed now? It wasn't needed before for svg assets right?

Reply:
Because this PR uses assets https://github.com/dfinity/gix-components/tree/style/new-design-colors-spacing-layout/src/lib/assets

Not svg encapsulated as svelte component but real assets which are `import ... from ....svg` with Vite and therefore are also resolved when the lib is packaged.